crates/egui-wgpu/src/renderer.rs

```diff
@@ -132,9 +132,10 @@ impl ScreenDescriptor {
 #[repr(C)]
 struct UniformBuffer {
     screen_size_in_points: [f32; 2],
+    pixels_per_point: f32,
```

Comment:
as per other comment, I don't see why this is needed

Reply:
Removed it now/turned it into the uniform controlling dithering.